target/linux/ipq40xx/image/generic.mk

```diff
@@ -1267,3 +1274,22 @@ define Device/zyxel_wre6606
 endef
 # Missing DSA Setup
 #TARGET_DEVICES += zyxel_wre6606
+
+define Device/zyxel_wsq50
+	DEVICE_VENDOR := ZyXEL
+	DEVICE_MODEL := WSQ50
+	DEVICE_ALT0_VENDOR := ZyXEL
+	DEVICE_ALT0_MODEL := WSQ60
+	SOC := qcom-ipq4019
+	KERNEL_SIZE := 4096k
+	BLOCKSIZE := 64k
+	RAS_BOARD := WSQ50
+	RAS_ROOTFS_SIZE := 20934k
+	RAS_VERSION := "$(VERSION_DIST) $(REVISION)"
+	DEVICE_PACKAGES := \
+		ath10k-firmware-qca4019-ct ath10k-firmware-qca9984-ct \
+		kmod-fs-ext4 kmod-mmc kmod-fs-squashfs tune2fs e2fsprogs \
```

Comment:
Remove `kmod-mmc` and `kmod-fs-squashfs` these are already compiled in the kernel

Reply:
This has been fixed - As before, superfluous packages removed